cmd/crypto/metadata_test.go

```diff
@@ -370,9 +370,9 @@ var isETagSealedTests = []struct {
 	ETag     string
 	IsSealed bool
 }{
-	{ETag: "", IsSealed: false},                                  // 0
-	{ETag: "90682b8e8cc7609c4671e1d64c73fc30", IsSealed: false},  // 1
-	{ETag: "f201040c9dc593e39ea004dc1323699bcd", IsSealed: true}, // 2 not valid ciphertext but looks like sealed ETag
+	{ETag: "", IsSealed: false},                                                                                                // 0
```

Comment:
Use go 1.11 locally the gofmt has changed this PR will fail ..

Reply:
Done :+1: